metacsv/core/internals.py

```diff
@@ -1006,14 +1009,14 @@ def to_netcdf(self, fp):
             <xarray.Dataset>
             Dimensions:  (index: 3)
             Coordinates:
-              * index    (index) int64 0 1 2
+              * index    (index) int32 0 1 2
```

Comment:
Looks like this is causing problems in tests. if the resulting data type is ambiguous maybe use an ellipsis.